project_testing/housing_characteristics/HVAC Shared Efficiencies.tsv

```diff
@@ -1,26 +1,3 @@
-Dependency=Heating Fuel	Dependency=HVAC Has Shared System	Option=Boiler Baseboards Heating Only, Electricity	Option=Boiler Baseboards Heating Only, Fuel Oil	Option=Boiler Baseboards Heating Only, Natural Gas	Option=Boiler Baseboards Heating Only, Propane	Option=Fan Coil Cooling Only	Option=Fan Coil Heating and Cooling, Electricity	Option=Fan Coil Heating and Cooling, Fuel Oil	Option=Fan Coil Heating and Cooling, Natural Gas	Option=Fan Coil Heating and Cooling, Propane	Option=None
```

Comment:
I don't know if these dependencies are needed in project_testing. May need to test.

Reply:
Integrity checks pass without these dependencies.